packages/google_maps_flutter/google_maps_flutter/android/src/main/java/io/flutter/plugins/googlemaps/GoogleMapController.java

```diff
@@ -18,6 +18,9 @@
 import androidx.lifecycle.DefaultLifecycleObserver;
 import androidx.lifecycle.Lifecycle;
 import androidx.lifecycle.LifecycleOwner;
+import androidx.savedstate.SavedStateRegistry;
```

Comment:
We need to add the savedstate dependency to the gradle file. IIUC it happens that we add the embedding dependencies as app-level dependencies which is why this currently build, but we actually don't want to allow apps to rely on the transitive dependencies (cc @blasten who I believe have some issue tracking this)

Reply:
Done